src/main/java/com/thegrizzlylabs/sardineandroid/impl/OkHttpSardine.java

```diff
@@ -105,7 +105,7 @@ public AuthenticationInterceptor(String userName, String password) {
 
         @Override
         public Response intercept(Chain chain) throws IOException {
-            Request request = chain.request().newBuilder().addHeader("Authorization", Credentials.basic(userName, password)).build();
+            Request request = chain.request().newBuilder().addHeader("Authorization", Credentials.basic(userName, password, SardineUtil.standardUTF8())).build();
```

Comment:
What happens if you don't specify any charset? Do you have a compilation error? A runtime error?

Reply:
It crashes the app on Android 4.x, because it will not find the StandardCharsets class, which does not exists on those version of Android, until API 19.

Reply:
Ok, I've just realized that StandardCharsets is not available before Android 19. This looks alright to me.